tests/subsys/logging/log_core/testcase.yaml

```diff
@@ -3,4 +3,4 @@ tests:
     tags: log_core logging
     platform_exclude: nucleo_l053r8 nucleo_f030r8
       stm32f0_disco native_posix native_posix_64 nrf52_bsim
-      qemu_riscv64
+      qemu_riscv64 up_squared_adsp
```

Comment:
the correct fix should be upstream now:
https://github.com/zephyrproject-rtos/zephyr/commit/d2dc7482294a8c934dde620202dbc1dfc59cc2b0